R/imports.R

```diff
@@ -1,6 +1,6 @@
 #' @importFrom dplyr arrange distinct filter first full_join group_by
-#' @importFrom dplyr inner_join last left_join mutate n_distinct rename
-#' @importFrom dplyr row_number select summarize tibble ungroup
+#' @importFrom dplyr inner_join last left_join right_join mutate n_distinct
+#' @importFrom dplyr row_number select summarize tibble ungroup rename
 #' @importFrom glue glue
 #' @importFrom rlang %||% abort warn
 #' @importFrom tidyr pivot_longer pivot_wider unnest
```

Comment:
It'd be nice to preserve the alpha order  (although rlang is messed up). In general I think it's not worth the effort -- unless we had a helper `r2dii.usethis::use_alpha_imports()` or similar (now that we are thinking of new features for r2dii.usethis).

https://github.com/2DegreesInvesting/r2dii.usethis/issues/35